src/db/tables/featureFlags.ts

```diff
@@ -38,17 +38,10 @@ export async function getDeletedFeatureFlags() {
 }
 
 export const featureFlagNames = [
-  "UpdatedEmailPreferencesOption",
   "CancellationFlow",
-  "ConfirmCancellation",
-  "FirstDataBrokerRemovalFixedEmail",
   "DiscountCouponNextThreeMonths",
   "LatestScanDateCsatSurvey",
-  "AutomaticRemovalCsatSurvey",
```

Comment:
I would prefer if we could keep all the feature flags related to the CSAT surveys. We only recently landed a PR that extends the functionality of this CSAT PR #5586.

Reply:
Just to make sure: when you say "keep all the feature flags", in the context of this PR you only mean undoing the removal of `AutomaticRemovalCsatSurvey`, right? I did that in 2cac1d137. Thanks for noticing.

Reply:
Thanks! Yes, I specifically mean “keep all the feature flags related to the CSAT surveys”.